Cargo.toml

```
@@ -33,6 +33,7 @@ base64 = "0.21.0"
 deno_ast = { version = "0.26.0", features = ["transpiling"] }
 deno_graph = "0.48.0"
 futures = "0.3.26"
+hashlink = "0.8.1"
```

```suggestion
hashlink = "0.8.2"
```
nit: 0.8.2 is the latest version